core/src/main/scala/kafka/server/BrokerServer.scala

```diff
@@ -113,7 +113,7 @@ class BrokerServer(
 
   var dynamicConfigHandlers: Map[String, ConfigHandler] = null
 
-  var replicaManager: ReplicaManager = null
+  @volatile private[this] var _replicaManager: ReplicaManager = null
```

Comment:
Could you make the same change for KafkaServer too?

Reply:
Done.